lib/PuppeteerSharp.Tests/PuppeteerTests/PuppeteerLaunchTests.cs

```diff
@@ -143,6 +141,85 @@ public class PuppeteerLaunchTests : PuppeteerBaseTest
             }
         }
 
+        [Fact]
+        public async Task UserDataDirOptionShouldRestoreState()
+        {
+            var launcher = new Launcher();
```

Comment:
this class inherits from `PuppeteerBaseTest` that creates a `Browser` for each test. maybe it should inherit from a more basic class that doesn't do anything since the created `Browser` isn't used?
I was thinking of adding a `PuppeteerBrowserBaseTest` that will work the same way `PuppeteerBaseTest` works now, and then refactor `PuppeteerBaseTest` to be more basic